custom-metrics-stackdriver-adapter/pkg/adapter/provider/translator.go

```diff
@@ -21,22 +21,21 @@ import (
 	"strings"
 	"time"
 
+	"github.com/GoogleCloudPlatform/k8s-stackdriver/custom-metrics-stackdriver-adapter/pkg/config"
+	"github.com/GoogleCloudPlatform/k8s-stackdriver/custom-metrics-stackdriver-adapter/pkg/provider"
+	"github.com/golang/glog"
 	stackdriver "google.golang.org/api/monitoring/v3"
+	"k8s.io/api/core/v1"
 	apierr "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
-	"k8s.io/client-go/pkg/api"
+	"k8s.io/kubernetes/pkg/api"
```

Comment:
all these `k8s.io/kubernetes` imports should be replaced with corev1 from k8s.io/api or with the scheme from the client.

Reply:
Done.